shared/helpers/macos/applescript.js

```diff
@@ -22,6 +22,9 @@ const APPLESCRIPT_TIMEOUT = 5;
 /** Error message to look for to determine error is an Applescript time out. */
 const APPLESCRIPT_TIMEOUT_ERROR = (exports.APPLESCRIPT_TIMEOUT_ERROR = 'AppleEvent timed out');
 
+const SHIFT_CHORD_DELAY = 0.05;
+const KEYPRESS_DELAY = 0.25;
```

Comment:
I somehow thought I'd have to ask you to pull these out as constants, but of course you're on top of it :)